### ISSUE TITLE
 DescriptionGroup (shortDescription, longDescription) must be embedded in mediaDescription

### ISSUE THREAD

**Issue by AlexErk** [ebu/radiodns-manager#132]
Validation error because of:

Validation	24, 43	cvc-complex-type.2.4.a:
Invalid content was found starting with element 'shortDescription‘.
One of '{"http://www.worlddab.org/schemas/spi/31":shortName, 
"http://www.worlddab.org/schemas/spi/31":longName,
"http://www.worlddab.org/schemas/spi/31":mediaDescription, 
"http://www.worlddab.org/schemas/spi/31":keywords, 
"http://www.worlddab.org/schemas/spi/31":link, 
"http://www.worlddab.org/schemas/spi/31":geolocation, WC[##other:"http://www.worlddab.org/schemas/spi/31"]}' is expected.

**Comment by magicbadger** [comment]
The `shortDescription` element is out of place, and should be contained within a `mediaDescription` element

e.g.

```
<mediaDescription>
  <shortDescription xml:lang="fr">B1 Obb </shortDescription>
</mediaDescription>
```
rather than
```
<shortDescription xml:lang="fr">B1 Obb </shortDescription>
```

**Comment by magicbadger** [comment]
closed in [release 4.1.1](https://github.com/ebu/radiodns-manager/releases/tag/4.1.1)
